data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java

```diff
@@ -103,12 +104,13 @@ private static void jettyEnableInherit(Server server) {
     @Bean
     public CommandLineRunner loadData(DataInitializer initializer) {
         return (args) -> {
-            initializer.initialize();
+            // initializer.initialize();
```

Comment:
This was actually commited by accident. I turned this off for testing. I did not intend to turn off fake data just yet.